### ISSUE TITLE
Custom recipes are not loaded in some cases

### ISSUE THREAD

**Issue by GSValore** [MoonManModding/EmergingTechnology#74]
## Version

1.3.9-beta (applies to current version as well)

## Problem

Custom recipes do not load on a linux-based server (or possibly system), it cannot find the file (the log file merely states this).

## Remedy

The current loaders for custom recipes use hard-coded path separators; to avoid this, an example of the following can be used instead (based on the custom-bulbs loader):

```java
String path = Paths.get(e.getModConfigurationDirectory().getAbsolutePath(), EmergingTechnology.MODID, "custom-bulbs.json").toString();
```

Paths.get() will allow the joining of paths while adhering to system separators.

**Comment by MoonManModding** [comment]
Hi,

Thanks for the bug report and the inclusion of a fix! The custom loaders were a bit rushed so I'll attempt to make them less brittle for the next release 👍

**Comment by MoonManModding** [comment]
This should be fixed from [version 1.3.13-beta](https://www.curseforge.com/minecraft/mc-mods/emerging-technology-hydroponics/files/2893142). If it still doesn't work correctly please let me know. Thanks 😁
